photon-lib/src/main/native/cpp/photonlib/RobotPoseEstimator.cpp

```diff
@@ -207,29 +211,32 @@ RobotPoseEstimator::ClosestToReferencePoseStrategy() {
         smallestDifference = alternativeDifference;
         pose = targetPose.TransformBy(
             target.GetAlternateCameraToTarget().Inverse());
-        milli = p.first->GetLatestResult().GetLatency() / 1000.;
+        stateTimestamp = p.first->GetLatestResult().GetTimestamp();
```

Comment:
This applies to other places too but GetLatestResult() will create a new packet every time. I'd rather we call GetLatestResult() exactly once, at the top of Update(), and reuse that result everywhere else?

Reply:
Ok, I really just suck at C++ lol

Reply:
The returned value of GetLatestResult() is unique to each camera, so it would be awkward to call it at the top of Update (we would have to store it somehow, etc.). There are a few places where GetLatestResult() is called for the same camera (like in this case). This could be solved by storing the value in a temporary variable. This would replace two GetLatestResult packet creations to one GetLatestResult packet creation and one storage into a variable. I believe this change won't have a large effect on the end performance and would complicate the code. Thoughts?